person.rb

```diff
@@ -1,21 +1,25 @@
-class Person
+require_relative 'nameable'
+
+class Person < Nameable
   attr_accessor :age, :name
   attr_reader :id
 
-  def initialize(_id, age, parent_permission: true, name: 'unknown')
+  def initialize(age, parent_permission: true, name: 'unknown')
+    super(name)
```

Comment:
- [ ] I am still facing the same error: 
![image](https://github.com/Metaverse-Nabeel/OOP-school-library/assets/62705947/97c81c1d-2fae-4fa7-bd2e-e8f1ce551d91)

Reply:
- How to fix it: name should be = to 'unknown' and not `:` also the name should not be on  the super but under it like the others, this way it would work check :arrow_down: 

```ruby
  def initialize(age, name = 'unknown', parent_permission: true)
    super()
    @id = Random.rand(1..1000)
    @name = name
    @age = age
    @parent_permission = parent_permission
  end
```

Reply:
Thank you so much @GzimAsani